drf_kit/models/__init__.py

```diff
@@ -29,6 +30,14 @@ class Meta(SoftDeleteModelMixin.Meta, OrderedModelMixin.Meta, BaseModel.Meta):
         indexes = SoftDeleteModelMixin.Meta.indexes + OrderedModelMixin.Meta.indexes + BaseModel.Meta.indexes
 
 
+class SoftDeleteAvailabilityModel(SoftDeleteModelMixin, AvailabilityModelMixin, BaseModel):
```

Comment:
Missing a test for `SoftDeleteAvailabilityModel`

Reply:
Thanks! I even found a bug when creating the test <3